Considerazioni valide per floss e proprietario. See: #70. (#71)
italia/guida-sviluppo-gestione-software-libero: per-chi-sviluppa-il-software/rilasciare-software-libero.rst

Rilasciare software libero
==========================

> Questo capitolo descrive delle buone pratiche per rilasciare software

Reviewer: Love these often underrated improvements BTW, small but fixing formalese constructs :heart:

in modo efficace seguendo il paradigma del *software* libero.

> Queste pratiche sono valide anche per tutti
> quei progetti che, con il software libero, condividono una o più
> delle seguenti caratteristiche:
> 
> - una aspettativa di vita medio/lunga, per cui la manutenzione del
>   codice è un aspetto importante;
> - un team di sviluppo che cambia nel tempo, per cui il tempo di
>   familiarizzazione con il codice è un aspetto importante.
> 

Reviewer: ```suggestion
```

Non lo metterei, le buone pratiche sono buone pratiche per ogni progetto e questo è tautologico.

Lo sviluppo di software proprietario è fuori ambito di questa guida, ma chi la volesse usare comunque come riferimento può :
* rilasciare il software come software libero :wink: 
* sostituire mentalmente durante la lettura le parti che non si applicano per il proprio caso

Author: > sono buone ... per ogni progetto

Explicit is better than implicit ;)

> Lo sviluppo di software proprietario è fuori ambito ...

Ogni progetto è potenzialmente open source. Chiarire che le indicazioni si applicano anche a progetti attualmente chiusi riduce le barriere alla transizione.

Inoltre, alcuni fornitori potrebbero suggerire agli enti che queste raccomandazioni non valgono per il software proprietario. Non credo quindi che l'indicazione sia tautologica.

I file da inserire nel repository
---------------------------------

> E' buona pratica che un repository contenga dei file che permettano

Reviewer: ```suggestion
È buona pratica che un repository contenga dei file che permettano
```

> al visitatore di familiarizzare col progetto, e in particolare capire:

Reviewer: ```suggestion
a chi lo visita di familiarizzare col progetto, e in particolare capire:
```

- se è il progetto che stanno cercando e se gli è utile;
- come iniziare a lavorare sul progetto nel minor tempo possibile

>   (e.g. entrare nel dettaglio la natura del progetto

Reviewer: Questa frase non quadra.

"Entrare nella natura del progetto"?
